demisto_sdk/commands/download/downloader.py

```diff
@@ -104,6 +104,29 @@ class ContentItemType(Enum):
 ]
 
 
+def format_playbook_task(content_item_data: dict[str, dict]) -> dict[str, dict]:
+    """This function checks if there are tasks that run sub-playbooks and converts 'playbookId' to 'playbookName' where applicable.
+       XSUP-39266: replacing playbookId with playbookName in tasks.
+
+    Args:
+        content_item_data (Dict): The content item data containing tasks.
+
+    Returns:
+        Dict: The updated content_item_data where 'playbookId' is replaced with 'playbookName' in tasks running sub-playbooks.
+    """
+    content_data = content_item_data.get("data", {})
+    all_tasks_data = content_data.get("tasks", {})
+    if isinstance(all_tasks_data, dict) and all_tasks_data:
+        for task_id, task_data in all_tasks_data.items():
+            playbook_id_value = task_data.get("task", {}).get("playbookId")
+            if task_data.get("type") == "playbook" and playbook_id_value:
+                task_data["task"]["playbookName"] = playbook_id_value
+                task_data["task"].pop("playbookId", None)
+                all_tasks_data[task_id] = task_data
+    content_data["tasks"] = all_tasks_data
+    return content_item_data
```

Comment:
not super critical as it works, but FYI:
```suggestion
            if task_data.get("type") == "playbook" and playbook_id_value:
                task_data["task"]["playbookName"] = task_data["task"].pop("playbookId")
```
no need to re-assign the `task_id` key to `task_data` - since `task_data` is a dict, its reference doesn't change even when we modify its content.

for the same reason:
1. no need to re-assign `content_data["tasks"] = all_tasks_data`
2. we don't need the method to return a value, we just modify the dict (the same rule is applied even if you are in a different method). Note: if you apply this change, make sure to change the call to `format_playbook_task` not to assign it to `value`.